### PR TITLE
CDSK-915 - add klog into addErrback

### PR DESCRIPTION
**Pull request by warcholprzemo**
Add klog into addErrback in `master` and `slave` directories

### CODE REVIEW

**Review by campos-ddc — APPROVED** [review]

File: master/buildbot/status/words.py [hunk 1]
```diff
@@ -822,7 +822,7 @@ def usageError(f):
             def logErr(f):
                 klog.err_json(f)
                 self.send("Something bad happened (see logs)")
-            d.addErrback(log.err)
+            d.addErrback(klog.err_json)
```
Comment (Mowinski) [inline master/buildbot/status/words.py]: I think you can remove this errorback. As you can see above, there is klog decorated by `addErrback`

**Review by Mowinski — COMMENTED** [review]
Did you consider to add klog to some global 'error handler'? I do not know it is possible, but in twisted should be place like global `try` - `catch`

File: master/buildbot/status/words.py (hunk 1 above)
Comment (warcholprzemo) [reply master/buildbot/status/words.py]: Done

**Comment by warcholprzemo** [comment]
I cannot find a ready solution for Global Error Handler (or something like that) for twisted. 
On the other hand monkey patching of twisted library, in my opinion, is not a good idea.

EDIT: There is twisted.python.log.DefaultObserver For now I cannot override it (setup into globals())

**Review by Mowinski — APPROVED** [review]

**Pull request merged**